Model/Builders/ConfigRepositoryBuilder.php

```diff
@@ -334,19 +335,21 @@ public function getPaymentNameByMethodCode($methodCode): string
     /**
      * {@inheritDoc}
      */
-    public function getError(array $transaction)
+    public function getError(Order $transaction)
     {
-        if ($transaction['status'] == 'error' && (current($transaction['transactions'])['customer_message']))
+        $status = $transaction->getStatus()->get();
+        $status = 'created';
+        if ($status == 'error' && $transaction->getCurrentTransaction()->getCustomerMessage())
         {
-            return current($transaction['transactions'])['customer_message'];
+            return $transaction->getCurrentTransaction()->getCustomerMessage();
         }
-        if (!empty($transaction['customer_messages']))
+//        if (!empty($transaction['customer_messages']))
+//        {
+//            return current($transaction['customer_messages'])["message"];
+//        }
```

Comment:
Previously, here was a check to see if `customer_messages` exists

Is such a check required for a plugin with sdk?